stdlib/public/core/Unicode.swift

```diff
@@ -451,14 +440,7 @@ public struct UTF16 : UnicodeCodec {
   public init() {}
 
   /// A lookahead buffer for one UTF-16 code unit.
-  internal var _decodeLookahead: UInt32 = 0
-
-  /// Flags with layout: `0b0000_00xy`.
-  ///
-  /// `y` is the EOF flag.
-  ///
-  /// `x` is set when `_decodeLookahead` contains a code unit.
-  internal var _lookaheadFlags: UInt8 = 0
+  internal var _decodeLookahead: UInt32?
```

Comment:
@gribozavr My bad, you're right. Would there be a difference though on 64-bit systems? 

Perhaps it's more consistent to make it a `UInt16?` or `CodePoint?`.